tests/wpt/metadata-layout-2020/css/css-ui/text-overflow-change-color.html.ini

```diff
@@ -1,2 +1,2 @@
 [text-overflow-change-color.html]
-  expected: FAIL
+  expected: CRASH
```

Comment:
This was an intermittent CRASH

```
  ▶ FAIL [expected CRASH] /css/css-ui/text-overflow-change-color.html
  │   → /css/css-ui/text-overflow-change-color.html ['46e943634bdce9b93ebb6cd319c2bcde3954a3d6']
  └   → /css/css-ui/reference/text-overflow-change-color-ref.html ['2bed3e2f829e307aab34676775b13e026f071663']
```

(Note: we need to update both layout-2020 and the 2013 metadata)